source/install/cloud-changelog.md

```diff
@@ -4,12 +4,38 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-07-27](#release-2022-07-27)
 - [Release 2022-07-20](#release-2022-07-20)
 - [Release 2022-07-13](#release-2022-07-13)
 - [Release 2022-06-29](#release-2022-06-29)
 - [Release 2022-06-22](#release-2022-06-22)
 - [Release 2022-06-15](#release-2022-06-15)
-- [Release 2022-06-14](#release-2022-06-14)
+
+## Release 2022-07-27
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to quickly and easily forward posts as permalinks with their respective permalink previews.
+ - Added a red destructive action color to ``Archive Channel`` and ``Leave Channel`` menu actions.
+ - Plugin activation errors now show in the plugin management page and marketplace.
+ - Added accessibility to the emoji picker skin tone selector and reversed the order of the skin tone selections in the emoji selector.
+
+#### Administration
+ - A new schema and API for audit logs was defined. Contrary to the previous audit log implementation, all audit log records now have the same schema.
+ - Added a new static system-level role called Custom Group Manager. This role has permissions to create, edit, and delete custom user groups via the existing front-end group management functionality. This functionality exists in Channels and can be used to assign individual users this ability when the mentioned Custom Groups permissions are removed for All Members via the **System Console** (**System Console > Permissions > Edit Scheme > Custom Groups**).
```

Comment:
@amyblais I made it even simpler, thanks for the quick changes:

Added a new static system-level role called Custom Group Manager. This role has permissions to create, edit, and delete custom user groups via **User Groups** in the Products menu. It can be used to assign individual users this ability when Custom Groups permissions are removed for All Members via the **System Console** (**System Console > Permissions > Edit Scheme > Custom Groups**).